### PR TITLE
Fg/doctor prompt

### PR DESCRIPTION
**Pull request by Gholi-MD**
added the llm_judge notebooks

### CODE REVIEW

File: reco_analysis/notebooks/read_JSON.ipynb
Comment (anniefriar-ucberkeley) [inline reco_analysis/notebooks/read_JSON.ipynb:1]: Would it be possible to clear output from this file before merging? That way the synthetic patient data is not in our repo. I accidentally did the same thing earlier by including patient data in my commit, but I think if we can keep it out of the repo that would be best. Thanks, and great work!

**Review by anniefriar-ucberkeley — COMMENTED** [review]
Great work Farid! I added one small comment. Otherwise it looks good to merge! Thanks!

File: reco_analysis/notebooks/read_JSON.ipynb
Comment (anniefriar-ucberkeley) [reply reco_analysis/notebooks/read_JSON.ipynb:1]: Upon review, I think all of our notebooks have done this. So more of a reminder for myself than anything!

Comment (Gholi-MD) [reply reco_analysis/notebooks/read_JSON.ipynb:1]: Thank you. Will aim to fix this in the next version

**Review by anniefriar-ucberkeley — APPROVED** [review]
Looks great!

**Pull request merged**